docker-compose.yaml

```diff
@@ -8,8 +8,7 @@ services:
       args:
         - GITHUB_PAT=${GITHUB_API_TOKEN}
     volumes:
-      - ./r:/src:cached # for watcher
-      - /src/renv/ # distinct renv in host/container
+      - ./r/R:/src/R:cached # for watcher
```

Comment:
I am not totally sure what these volumes were/are for but I know it works, so I'll approve and maybe you can explain at your leisure

Reply:
Basically they are for the watcher so that the worker will restart if the source code changes. The first mounted a volume into the r folder and the second was supposed to prevent the `renv/` folder from also being populated from the host. It didn't seem to have the expected behavior so instead I just direct mount into the `./r/R` folder where _most_ of the worker source code that we want to watch is.

Reply:
Oh, this explains why editing `work.R` didn't trigger a restart when I tried